### PR TITLE
Travis CI: add build configuration for Android NDK with cmake.

### PR DESCRIPTION
**Pull request by antiagainst**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,6 +25,8 @@ compiler:
   - gcc
 
 matrix:
+  include:
+    - nev: BUILD_NDK=ON
```
Comment (dneto0) [inline .travis.yml]: What does "nev" mean here?

Comment (antiagainst) [inline .travis.yml]: Ah, sorry, kinda forgot about this PR. Fixed.

Comment (antiagainst) [inline .travis.yml]: Actually should be `env` to set environment variable.

**Comment by dneto0** [comment]
+2

**Comment by antiagainst** [comment]
Merged as e96a881b0bb7e42251566e12b8788134c9917aaf.

**Pull request closed without merging**